extension/js/removeNestedQuotes.js

```diff
@@ -33,3 +33,62 @@ if (onlyLastQuote) {
 		messageArea.innerHTML = text;
 	}
 }
+
+function lastaddquote(post_id, username, l_wrote) {
+	var message_name = 'message_' + post_id;
+	var theSelection = '';
+	var divarea = false;
+	var i;
+
+	if (l_wrote === undefined) {
+		// Backwards compatibility
+		l_wrote = 'wrote';
+	}
+
+	if (document.all) {
+		divarea = document.all[message_name];
+	} else {
+		divarea = document.getElementById(message_name);
+	}
+
+	// Get text selection - not only the post content :(
+	// IE9 must use the document.selection method but has the *.getSelection so we just force no IE
+	if (window.getSelection && !is_ie && !window.opera) {
+		theSelection = window.getSelection().toString();
+	} else if (document.getSelection && !is_ie) {
+		theSelection = document.getSelection();
+	} else if (document.selection) {
+		theSelection = document.selection.createRange().text;
+	}
```

Comment:
why do you need this? why not just use `getElementById`?
`document.all` is Deprecated